
<!DOCTYPE html PUBLIC -//W3C//DTD XHTML 1.0 Strict//EN http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd>
<html xmlns=http://www.w3.org/1999/xhtml xml:lang=en lang=en>
<head>
<meta http-equiv=content-type content=text/html;charset=iso-8859-2 />
<meta name=author content=free-css-templates.com />
<link rel=stylesheet href=./../HPS_INPUTS/REPORT/stylesheet.css type=text/css />
<title>HPS Research - Dr. Nelson R. Manohar</title>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<div id=content>
<div id=title> <h1> <img src=./../HPS_INPUTS/REPORT/IMAGES/cat.jpg height=80 alt=hpslogo /> <span> HPS DNA MINING SYSTEMS RESEARCH - <a href=HPS_FOREWORD.HTM> <span class=green> DR. NELSON R.
MANOHAR </span> </a> </span> </h1> <span id=slogan> HPS DNA MINING SYSTEMS (c) 2007 BY NELSON R. MANOHAR-ALERS, PH.D.  </span> </div>

<div id=menu>
<div class=submit>
<ul>
<li><a href=HPS_FOREWORD.HTM class=unselected>                 <span><b>FOREWORD</b></span></a></li>
<li><a href=HPS_INTRODUCTION.HTM class=unselected>             <span><b>INTRO</b></span></a></li>
<li><a href=HPS_INPUTS.HTM class=unselected>                   <span><b>INPUTS</b></span></a></li>
<li><a href=HPS_DATA_DESCRIPTION.HTM class=unselected>         <span><b>OPTIONAL</b></span></a></li>
<li><a href=HPS_BEHAVIOR.HTM class=unselected>                 <span><b>PERF</b></span></a></li>
<li><a href=HPS_MATCHING_INSTANCES.HTM class=unselected>       <span><b>MATCHES</b></span></a></li>
<li><a href=HPS_RECONSTRUCTION_OPS.HTM class=unselected>       <span><b>REPAIR</b></span></a></li>
<li><a href=HPS_SIMILARITY_PLOTS.HTM class=unselected>         <span><b>SIMILARITY</b></span></a></li>
<li><a href=HPS_PHILOGRAM_PLOT.HTM class=selected>           <span><b>PHILOGRAM</b></span></a></li>
<li><a href=HPS_MOTIF_SEARCH.HTM class=unselected>             <span><b>MOTIFS</b></span></a></li>
<li><a href=HPS_MOTIF_REGEXPR.HTM class=unselected>            <span><b>MOTIF-MINER</b></span></a></li>
<li><a href=HPS_ABOUT_US.HTM class=unselected>                 <span><b>ABOUT</b></span></a></li>
<li><a href=HPS_DISCLAIMER.HTM class=unselected>               <span><b>DISCLAIMER</b></span></a></li>
</ul>
</div>
</div>


<div id=subheader>
<div class=rside>
Auto-generated report for multiple sequence alignment (MSA) analysis of the DNA signature against DNA test sequence.
</div>

<div class=lside>
<strong> Welcome! </strong> 
</div>
</div>

<div id=maincontent>
<A HREF=http://www3.webng.com/nelsonmanohar/research.htm> <P align=center> <B> <EM> (c) Nelson R. Manohar, Ph.D. - 2005-2007. All rights are reserved. </EM> </B> </span> </A> 

<div id=right_side>
<div class=lcontent align=justify> <h3> SITE MAP FOR AUTO-GENERATED REPORT</h3> 
<BR>   <a href=HPS_FOREWORD.HTM>                 <span><b>FOREWORD</b></span></a> 	 - important remarks.</li>
<BR>   <a href=HPS_INTRODUCTION.HTM>             <span><b>INTRODUCTION</b></span></a>    - to HPS DNA Mining Systems.</li>
<BR>   <a href=HPS_INPUTS.HTM>                   <span><b>INPUTS</b></span></a> 	 - description of the processed inputs.</li>
<BR>   <a href=HPS_DATA_DESCRIPTION.HTM>         <span><b>OPTIONAL</b></span></a> 	 - user-provide description of inputs.</li>
<BR>   <a href=HPS_BEHAVIOR.HTM>                 <span><b>PERFORMANCE</b></span></a>     - performance of dna mining.</li>
<BR>   <a href=HPS_MATCHING_INSTANCES.HTM>       <span><b>MATCHES</b></span></a>         - overview of matching instances.</li>
<BR>   <a href=HPS_RECONSTRUCTION_OPS.HTM>       <span><b>RECONSTRUCTION</b></span></a>  - summary of matching instances.</li>
<BR>   <a href=HPS_SIMILARITY_PLOTS.HTM>         <span><b>SIMILARITY</b></span></a>      - before/after dot matrix plots.</li>
<BR>   <a href=HPS_PHILOGRAM_PLOT.HTM>           <span><b>PHILOGRAM</b></span></a>       - signature-rooted philogram.</li>
<BR>   <a href=HPS_MOTIF_SEARCH.HTM>             <span><b>MOTIF-ID</b></span></a>        - motif characterization tool.</li>
<BR>   <a href=HPS_MOTIF_REGEXPR.HTM>            <span><b>MOTIF-MINER</b></span></a>     - motif miner tool.</li>
<BR>   <a href=HPS_ABOUT_US.HTM>                 <span><b>ABOUT USR</b></span></a>       - about us.</li>
<BR>   <a href=HPS_DISCLAIMER.HTM>               <span><b>DISCLAIMER</b></span></a>      - important remarks.</li>
</div>
</div>

<div id=left_side align=justify>



<HTML> <HEAD> <TITLE>HPS DNA MINING SYSTEMS</TITLE> <STYLE>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<H3> DNA SIGNATURE-ROOTED PHYLOGRAM </H3>
<P> 	As stated, all matching instances are TRUE matching instances of the DNA signature, i.e., the limiting probability 
	of (such matching instance being a TRUE match to the DNA signature) is ONE in the formal sense (i.e., virtual 
	certainty for all practical purposes).  Therefore, the distance metrics used in this plot rather than representing 
	measurements of the probabilistic fitness of any such match represent instead abstraction measures of the DNA 
	damage that is accounted for within a matching instance. As shown above, in the reconstruction edit plots, it has 
	been determined that after such specified DNA reconstruction, all such burst DNA damage could be corrected and an 
	exacting reproduction of the original DNA signature is therefore achievable from the matching instance. 
</P>
<H4> RELATIVE EVOLUTIONARY COST METRICS </H4>
<P> 	In the following plot, points represent the exacting (x,y,z) cost-metric positioning of the given matching instance,
	The line shown traverses the set of matching instances along a sorted partial order based on the total number of 
	uncorrectable errors (as well as some other cost-metrics).  This way, the DNA signature is found at the (0,0,0) 
	coordinates and each point along the line represents a matching instance of greater number of accumulated 
	uncorrectable errors.  In actuality, the evolutionary cost-metric used to generate the plot is a three-dimensional 
	cost metric and thus this ordering is a partial order in the formal sense. As a result, the positioning (x2, y2, z2) 
	of the subsequent (partially ordered) matching instance is found to be in (and displayed as) a proportional 
	delta with respect to the previous (x1, y1, z1) coordinates.  This way, edges represent an estimate of the 
	total evolutionary 3D-cost-metric change between consecutive matching instances.  Note that the distance from 
	the (0,0,0)=DNA-SIGNATURE vertex to the actual (x,y,z) coordinate of any matching instance represents a form of 
	absolute evolutionary distance.  Similarly, the lenght of an edge between consecutive nodes simply allows computing 
	the magnitude of their internodal evolutionary distance by a straightforward application of the Phytagoras' theorem. 
</P>
<H4> PHYLOGENETIC SORTING </H4>
<P> 	Note that, in the pure sense, the following plot does NOT represent a PHYLOGENETIC TREE as edges between nodes (i.e., 
	matching instances) relate differential cost-metrics among matching instances (as well as with respect to the DNA 
	signature) as opposed to evolutionary relationships between matching instances.  Note however, that the plot can be 
	used as a STARTING POINT to derive a phylogenetic tree between the matching instances and the DNA signature. 
</P>
<TABLE WIDTH=1200 BGCOLOR=#E6F2FF ALIGN=JUSTIFY>
<TH> PHYLOGENETIC SORTING </TH>
<TR> <TD> <IMG SRC=HPS_DNA_PHYLOGENIC_TREEPLOT.PNG WIDTH=1200 HEIGHT=800 HSPACE=0 VSPACE=0 BORDER=0 ALIGN=JUSTIFY> </TD> </TR> <TR>
<TD AlIGN=TOP> <SPAN STYLE='font-size:9.0pt;font-family:Verdana'> <DL> 
<DT><STRONG>BOT</STRONG> Distance 3D metric relationship between the DNA signature and unearthed matching instances. This plot 
can be used to derive a phylogenetic tree showing the evolutionary relationship between the DNA signature and matching instances. 
</DT> </DL> </SPAN> </TD> </TR> </TABLE>
</BODY> </HTML>



</div>
</div>

<div id=footer>
<p class=right> NELSON R. MANOHAR &copy; 2007  &middot; Updated on 5/07 &middot; Design: 
<a href=http://www.free-css-templates.com title=Designed by free-css-templates.com>free-css-templates.com</a> &middot;
</p>
</div> 
</div> </body> </html>
