
<!DOCTYPE html PUBLIC -//W3C//DTD XHTML 1.0 Strict//EN http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd>
<html xmlns=http://www.w3.org/1999/xhtml xml:lang=en lang=en>
<head>
<meta http-equiv=content-type content=text/html;charset=iso-8859-2 />
<meta name=author content=free-css-templates.com />
<link rel=stylesheet href=./../HPS_INPUTS/REPORT/stylesheet.css type=text/css />
<title>HPS Research - Dr. Nelson R. Manohar</title>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<div id=content>
<div id=title> <h1> <img src=./../HPS_INPUTS/REPORT/IMAGES/cat.jpg height=80 alt=hpslogo /> <span> HPS DNA MINING SYSTEMS RESEARCH - <a href=HPS_FOREWORD.HTM> <span class=green> DR. NELSON R.
MANOHAR </span> </a> </span> </h1> <span id=slogan> HPS DNA MINING SYSTEMS (c) 2007 BY NELSON R. MANOHAR-ALERS, PH.D.  </span> </div>

<div id=menu>
<div class=submit>
<ul>
<li><a href=HPS_FOREWORD.HTM class=unselected>                 <span><b>FOREWORD</b></span></a></li>
<li><a href=HPS_INTRODUCTION.HTM class=unselected>             <span><b>INTRO</b></span></a></li>
<li><a href=HPS_INPUTS.HTM class=unselected>                   <span><b>INPUTS</b></span></a></li>
<li><a href=HPS_DATA_DESCRIPTION.HTM class=selected>         <span><b>OPTIONAL</b></span></a></li>
<li><a href=HPS_BEHAVIOR.HTM class=unselected>                 <span><b>PERF</b></span></a></li>
<li><a href=HPS_MATCHING_INSTANCES.HTM class=unselected>       <span><b>MATCHES</b></span></a></li>
<li><a href=HPS_RECONSTRUCTION_OPS.HTM class=unselected>       <span><b>REPAIR</b></span></a></li>
<li><a href=HPS_SIMILARITY_PLOTS.HTM class=unselected>         <span><b>SIMILARITY</b></span></a></li>
<li><a href=HPS_PHILOGRAM_PLOT.HTM class=unselected>           <span><b>PHILOGRAM</b></span></a></li>
<li><a href=HPS_MOTIF_SEARCH.HTM class=unselected>             <span><b>MOTIFS</b></span></a></li>
<li><a href=HPS_MOTIF_REGEXPR.HTM class=unselected>            <span><b>MOTIF-MINER</b></span></a></li>
<li><a href=HPS_ABOUT_US.HTM class=unselected>                 <span><b>ABOUT</b></span></a></li>
<li><a href=HPS_DISCLAIMER.HTM class=unselected>               <span><b>DISCLAIMER</b></span></a></li>
</ul>
</div>
</div>


<div id=subheader>
<div class=rside>
Auto-generated report for multiple sequence alignment (MSA) analysis of the DNA signature against DNA test sequence.
</div>

<div class=lside>
<strong> Welcome! </strong> 
</div>
</div>

<div id=maincontent>
<A HREF=http://www3.webng.com/nelsonmanohar/research.htm> <P align=center> <B> <EM> (c) Nelson R. Manohar, Ph.D. - 2005-2007. All rights are reserved. </EM> </B> </span> </A> 

<div id=right_side>
<div class=lcontent align=justify> <h3> SITE MAP FOR AUTO-GENERATED REPORT</h3> 
<BR>   <a href=HPS_FOREWORD.HTM>                 <span><b>FOREWORD</b></span></a> 	 - important remarks.</li>
<BR>   <a href=HPS_INTRODUCTION.HTM>             <span><b>INTRODUCTION</b></span></a>    - to HPS DNA Mining Systems.</li>
<BR>   <a href=HPS_INPUTS.HTM>                   <span><b>INPUTS</b></span></a> 	 - description of the processed inputs.</li>
<BR>   <a href=HPS_DATA_DESCRIPTION.HTM>         <span><b>OPTIONAL</b></span></a> 	 - user-provide description of inputs.</li>
<BR>   <a href=HPS_BEHAVIOR.HTM>                 <span><b>PERFORMANCE</b></span></a>     - performance of dna mining.</li>
<BR>   <a href=HPS_MATCHING_INSTANCES.HTM>       <span><b>MATCHES</b></span></a>         - overview of matching instances.</li>
<BR>   <a href=HPS_RECONSTRUCTION_OPS.HTM>       <span><b>RECONSTRUCTION</b></span></a>  - summary of matching instances.</li>
<BR>   <a href=HPS_SIMILARITY_PLOTS.HTM>         <span><b>SIMILARITY</b></span></a>      - before/after dot matrix plots.</li>
<BR>   <a href=HPS_PHILOGRAM_PLOT.HTM>           <span><b>PHILOGRAM</b></span></a>       - signature-rooted philogram.</li>
<BR>   <a href=HPS_MOTIF_SEARCH.HTM>             <span><b>MOTIF-ID</b></span></a>        - motif characterization tool.</li>
<BR>   <a href=HPS_MOTIF_REGEXPR.HTM>            <span><b>MOTIF-MINER</b></span></a>     - motif miner tool.</li>
<BR>   <a href=HPS_ABOUT_US.HTM>                 <span><b>ABOUT USR</b></span></a>       - about us.</li>
<BR>   <a href=HPS_DISCLAIMER.HTM>               <span><b>DISCLAIMER</b></span></a>      - important remarks.</li>
</div>
</div>

<div id=left_side align=justify>



<HTML> <HEAD> <TITLE>HPS DNA MINING SYSTEMS</TITLE> <STYLE>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<H3>DESCRIPTION OF THE TEST DATA</H3>
<P> 	<B> THIS </B> SECTION WAS NOT AUTO-GENERATED. It is provided for convenience of those 
	seeking to review, rate, or compare the results presented herein.  To test the HPS DNA 
	pattern miner, a DNA input sequence of roughly 20000 base pairs (bp) was constructed 
	using the 13000 bp E-COLI DNA sequence 1786520 (obtained from ENTREZ) as a starting point. 
	Said sequence was then MODIFIED as follows.  First, the DNA signature 
	was selected to be simply the bases contained within the genomic addresses 5000 TO 5999. 
	Then, to test several key conditions, various <B> damaged instances of the DNA signature </B> 
	were manufactured (as described below) and INSERTED INTO SPECIFIC LOCATIONS.
	This augmentation process is described below. AS SHOWN, ALL TEST CASES WERE CORRECTLY UNEARTHED, RATED, AND REPORTED.
</P>
<H4>LAYOUT OF TEST-CASE MATCHING INSTANCES </H4>
<TABLE CLASS=MSONORMAL ALIGN=JUSTIFY>
		   <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
		   	Between [5000:6000]
		   </SPAN> </TD> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
		   	the <B> identical matching instance </B> of the DNA signature is found.
		   </SPAN> </TD> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	 	   	The original DNA test sequence 1786520 had 13480 bp.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [7000:8000] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
		   	a <B> heavily single-point mutated version </B> of the original DNA signature was inserted, 
			consisting of approximately pseudo-random 25 single-point mutations from A->T coupled with
			approximately 25 single-point mutations from T->A. The mutations were chosen to take place at
			particular indexes (e.g., every 10th base which happened to be A) to easily verify via 
			eye inspection of proper detection within the matching instance. Effectively, the resulting 
			matching instance had about 5% (50/1000) randomly single-point mutations which HPS algorithms
			would have to deal with in the identification of the intrinsic structure of the underlying 
			DNA signature found within. 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
		   	This transformation increased the resultant DNA test sequence in size from 13480 to 14480.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [13481:14450] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> heavily damaged version of the DNA signature </B> was inserted, which exhibited a 
			SINGLE DELETION BURST of exactly 30 base pairs at (intra) genomic address (800). To accomplish
			this, a true DNA signature match was inserted therein, but then basepairs at (DNA signature) 
			genomic addresses [800:830] were deleted. No single point mutations were inflicted onto this 
			true matching instance. 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the resultant DNA test sequence in size fro 14480 to 15550.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [14550:17000] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> DNA filler </B> sequence extracted from a subsequence of the original DNA test sequence
			(but OUTSIDE of the DNA signature region) was inserted into the DNA test sequence.  
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 15550 to 17000.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [17000:18000] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> heavily damaged version of the DNA signature </B> was inserted, which exhibited (1) a DELETION 
			BURST of exactly 30 base pairs at (intra) genomic address (300) coupled with (2) a RANDOM INSERTION 
			BURST of 30 base pairs at (intra) genomic address (800). Effectively, a true DNA signature match was 
			inserted therein, but with deleted basepairs at genomic addresses [300:330] and inserted RANDOM 
			bases at genomic addresses [800:830]. No single point mutations were inflicted onto this true 
			matching instance. 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 17000 to 18000.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [18000:19000] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			another <B> DNA filler </B> sequence extracted from a subsequence of the original DNA test sequence
			(but OUTSIDE of the DNA signature region) was inserted into the DNA test sequence.  
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 18000 to 19000.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [19000:19970] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> heavily damaged version of the DNA signature </B> was inserted, which exhibited a SINGLE DELETION 
			BURST of exactly 30 base pairs at (intra) genomic address (300). Effectively, a true DNA signature 
			match was inserted therein, but with deleted basepairs at genomic addresses [300:330]. No single 
			point mutations were inflicted onto this true matching instance. 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 19000 to 20000.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [19970:30050] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> DNA filler </B> sequence extracted from SEQUENCE_1786454.DNA (10080 bp) was inserted.
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 20000 to 30000.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [24000:24975] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> heavily damaged version of the DNA signature </B> was inserted, which exhibited (1) multiple DELETION 
			BURSTs of exactly 8 base pairs at every 200 (intra) genomic address (i.e., 200, 400, 600, 800) and then 
			coupled with (2) a single INSERTION BURST of 8 base pairs at (intra) genomic address (500).  No single point 
			mutations were inflicted onto this true matching instance. 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 30000 to 31000.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [30000:30975] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> heavily damaged version of the DNA signature </B> was inserted, which exhibited the above DNA damage
			coupled with several random single point mutations at periodic intervals (1 base every 50 bases, for the 
			first 500 genomic addresses). 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 31000 to 32000.
		   </SPAN> </TD> </TR> <TR> <TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
	           	Between [32002:32978] 
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			a <B> heavily damaged version of the DNA signature </B> was inserted, which exhibited multiple DELETION 
			BURSTs of exactly 8 base pairs at every 200 (intra) genomic address (i.e., 200, 400, 600, 800).
		   </SPAN> </TD><TD> <SPAN STYLE='font-size:8.0pt;font-family:Verdana'>
			This increased the size from 32000 to 33000. Total input size (approx. 1000+33000bp).
		   </SPAN> </TD> </TR>
</TABLE>
</BODY> </HTML> 


<BR>
</div>
</div>

<div id=footer>
<p class=right> NELSON R. MANOHAR &copy; 2007  &middot; Updated on 5/07 &middot; Design: 
<a href=http://www.free-css-templates.com title=Designed by free-css-templates.com>free-css-templates.com</a> &middot;
</p>
</div> 
</div> </body> </html>
