
<!DOCTYPE html PUBLIC -//W3C//DTD XHTML 1.0 Strict//EN http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd>
<html xmlns=http://www.w3.org/1999/xhtml xml:lang=en lang=en>
<head>
<meta http-equiv=content-type content=text/html;charset=iso-8859-2 />
<meta name=author content=free-css-templates.com />
<link rel=stylesheet href=./../HPS_INPUTS/REPORT/stylesheet.css type=text/css />
<title>HPS Research - Dr. Nelson R. Manohar</title>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<div id=content>
<div id=title> <h1> <img src=./../HPS_INPUTS/REPORT/IMAGES/cat.jpg height=80 alt=hpslogo /> <span> HPS DNA MINING SYSTEMS RESEARCH - <a href=HPS_FOREWORD.HTM> <span class=green> DR. NELSON R.
MANOHAR </span> </a> </span> </h1> <span id=slogan> HPS DNA MINING SYSTEMS (c) 2007 BY NELSON R. MANOHAR-ALERS, PH.D.  </span> </div>

<div id=menu>
<div class=submit>
<ul>
<li><a href=HPS_FOREWORD.HTM class=unselected>                 <span><b>FOREWORD</b></span></a></li>
<li><a href=HPS_INTRODUCTION.HTM class=unselected>             <span><b>INTRO</b></span></a></li>
<li><a href=HPS_INPUTS.HTM class=unselected>                   <span><b>INPUTS</b></span></a></li>
<li><a href=HPS_DATA_DESCRIPTION.HTM class=unselected>         <span><b>OPTIONAL</b></span></a></li>
<li><a href=HPS_BEHAVIOR.HTM class=unselected>                 <span><b>PERF</b></span></a></li>
<li><a href=HPS_MATCHING_INSTANCES.HTM class=unselected>       <span><b>MATCHES</b></span></a></li>
<li><a href=HPS_RECONSTRUCTION_OPS.HTM class=unselected>       <span><b>REPAIR</b></span></a></li>
<li><a href=HPS_SIMILARITY_PLOTS.HTM class=unselected>         <span><b>SIMILARITY</b></span></a></li>
<li><a href=HPS_PHILOGRAM_PLOT.HTM class=unselected>           <span><b>PHILOGRAM</b></span></a></li>
<li><a href=HPS_MOTIF_SEARCH.HTM class=selected>             <span><b>MOTIFS</b></span></a></li>
<li><a href=HPS_MOTIF_REGEXPR.HTM class=unselected>            <span><b>MOTIF-MINER</b></span></a></li>
<li><a href=HPS_ABOUT_US.HTM class=unselected>                 <span><b>ABOUT</b></span></a></li>
<li><a href=HPS_DISCLAIMER.HTM class=unselected>               <span><b>DISCLAIMER</b></span></a></li>
</ul>
</div>
</div>


<div id=subheader>
<div class=rside>
Auto-generated report for multiple sequence alignment (MSA) analysis of the DNA signature against DNA test sequence.
</div>

<div class=lside>
<strong> Welcome! </strong> 
</div>
</div>

<div id=maincontent>
<A HREF=http://www3.webng.com/nelsonmanohar/research.htm> <P align=center> <B> <EM> (c) Nelson R. Manohar, Ph.D. - 2005-2007. All rights are reserved. </EM> </B> </span> </A> 

<div id=right_side>
<div class=lcontent align=justify> <h3> SITE MAP FOR AUTO-GENERATED REPORT</h3> 
<BR>   <a href=HPS_FOREWORD.HTM>                 <span><b>FOREWORD</b></span></a> 	 - important remarks.</li>
<BR>   <a href=HPS_INTRODUCTION.HTM>             <span><b>INTRODUCTION</b></span></a>    - to HPS DNA Mining Systems.</li>
<BR>   <a href=HPS_INPUTS.HTM>                   <span><b>INPUTS</b></span></a> 	 - description of the processed inputs.</li>
<BR>   <a href=HPS_DATA_DESCRIPTION.HTM>         <span><b>OPTIONAL</b></span></a> 	 - user-provide description of inputs.</li>
<BR>   <a href=HPS_BEHAVIOR.HTM>                 <span><b>PERFORMANCE</b></span></a>     - performance of dna mining.</li>
<BR>   <a href=HPS_MATCHING_INSTANCES.HTM>       <span><b>MATCHES</b></span></a>         - overview of matching instances.</li>
<BR>   <a href=HPS_RECONSTRUCTION_OPS.HTM>       <span><b>RECONSTRUCTION</b></span></a>  - summary of matching instances.</li>
<BR>   <a href=HPS_SIMILARITY_PLOTS.HTM>         <span><b>SIMILARITY</b></span></a>      - before/after dot matrix plots.</li>
<BR>   <a href=HPS_PHILOGRAM_PLOT.HTM>           <span><b>PHILOGRAM</b></span></a>       - signature-rooted philogram.</li>
<BR>   <a href=HPS_MOTIF_SEARCH.HTM>             <span><b>MOTIF-ID</b></span></a>        - motif characterization tool.</li>
<BR>   <a href=HPS_MOTIF_REGEXPR.HTM>            <span><b>MOTIF-MINER</b></span></a>     - motif miner tool.</li>
<BR>   <a href=HPS_ABOUT_US.HTM>                 <span><b>ABOUT USR</b></span></a>       - about us.</li>
<BR>   <a href=HPS_DISCLAIMER.HTM>               <span><b>DISCLAIMER</b></span></a>      - important remarks.</li>
</div>
</div>

<div id=left_side align=justify>



<HTML> <HEAD> <TITLE>HPS DNA MINING SYSTEMS</TITLE> <STYLE>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<H3> SEQUENCE MOTIF FINDER </H3>
<P>	"HPS DNA mining systems" provide additional functions such as the <B> sequence motif 
	finder. </B> The sequence motif finder is a sequence-motif miner and visualizer 
	that allows identifying the relative frequency and location for any, some, or even all
	DNA sequence-motifs DUETS, TRIPLETS, and/or QUARTETS (i.e., 2, 3, and 4-base).  
	The sequence motif finder has linear run-time costs. For your convenience, 
   	"HPS DNA mining systems" applies the sequence-motif finder to the DNA signature 
	by default - unless otherwise you specified a different subsequence interval. 
</P>
<H4> SUMMARY INFORMATION FOR MOTIF DUETS, TRIPLETS, AND QUARTETS </H4>
<P> 	The figure below shows the top (most frequent) sequence-motif DUETS, TRIPLETS, 
	and QUARTETS found to lie within the specified subsequence of the DNA input sequence. 
	As stated, the full extent of the DNA signature is examined by default, although it 
	is possible to examine ANY subsequence of the <B> DNA input sequence. </B> Therefore, 
	the subsequence interval analyzed is identified in the plot by a range notation of 
	the form [#:#].  To allow closer examination, the selected interval of the DNA input 
	sequence has been automatically divided into four subintervals of similar size being 
	ordered (by increasing genomic address) from top to bottom.
	Summary data along the top part of the figure provides information 
	about the relative frequency and identity of sequence-motifs found based on top-most 
	frequency rank. Summary data is divided into four columns that corresponds (from left 
	to right) to: (1) summary data for the top 5 motif DUETS,  (2) summary data for the 
	top 5 motif TRIPLETS, (3) summary data for the top 5 motif QUARTETS, and (4) summary 
	data for the 4 DNA bases.  By default, the top 5 sequence-motifs duets, triplets, and 
	quartets are analyzed; however, this number can be specified.  Moreover, for each 
	sequence-motif, a <B> tight-bound interval </B> describes the number of <B> motif-repeats 
	</B> found for said sequence-motif.  In practicality, the lower-estimate (of the interval 
	given) represents the true number of motif-repeats that corresponds to a sequence-motif. 
	Furthermore, for each sequence-motif found subsequence, a color-code is assigned (note 
	however that colors are re-used across columns).  The motif's assigned color is used to 
	color-code the display of <B> all repeats of its corresponding sequence-motif. </B> This 
	way, each sequence-motif is shown by a stream of color-coded vertical arrows/lines placed 
	in exactly <B> one row per sequence-motif. </B> Normally, all arrow-rows for sequence-motif 
	DUETS are printed first, then all arrow-rows for TRIPLETS are printed, and finally, all 
	arrow-rows for QUARTETS are printed.  However, by default, the display of said motif-repeats 
	arrow-rows is shown <B> only for the topmost sequence-motif </B> each for DUETS (top row), 
	TRIPLETS (middle row), and QUARTETS (bottom row). The number of arrow-rows to print for 
	the resultant DUETS, TRIPLETS, AND QUARTETS motif-set can be specified and its limited only 
	by physical constraints of the plot.
</P>
<TABLE WIDTH=1200 BGCOLOR=#E6F2FF ALIGN=JUSTIFY>
<TH> SEQUENCE MOTIF FINDER </TH>
<TR> <TD> <IMG SRC=HPS_DNA_SEQUENCE_ANALYZER_PLOT.PNG WIDTH=1200 HEIGHT=800 HSPACE=0 VSPACE=0 BORDER=0 ALIGN=JUSTIFY> 
</TD> </TR> <TR> <TD AlIGN=TOP> <SPAN STYLE='font-size:9.0pt;font-family:Verdana'> <DL> 
<DT> Repeats of the highest frequency sequence-motif(s) found to lie within the specified subsequence. Default
setup displays summary (frequency and identify) data for only the TOP FIVE FREQUENCY sequence-motif DUETs, TRIPLETs, 
and QUARTETs (i.e., 2, 3, and 4 DNA-base sequences).  The relative location for each of the motif-repeats of the 
TOPMOST DUET, TRIPLET, and QUARTET is also shown by means of color-coded arrow-rows. Both the number of (2, 3, 4-base) 
motif-repeat arrow-rows to display and the number of sequence-motifs to rank can be specified. </DT>
</DL> </SPAN> </TD> </TR> </TABLE>
</BODY> </HTML>



</div>
</div>

<div id=footer>
<p class=right> NELSON R. MANOHAR &copy; 2007  &middot; Updated on 5/07 &middot; Design: 
<a href=http://www.free-css-templates.com title=Designed by free-css-templates.com>free-css-templates.com</a> &middot;
</p>
</div> 
</div> </body> </html>
