
<!DOCTYPE html PUBLIC -//W3C//DTD XHTML 1.0 Strict//EN http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd>
<html xmlns=http://www.w3.org/1999/xhtml xml:lang=en lang=en>
<head>
<meta http-equiv=content-type content=text/html;charset=iso-8859-2 />
<meta name=author content=free-css-templates.com />
<link rel=stylesheet href=./../HPS_INPUTS/REPORT/stylesheet.css type=text/css />
<title>HPS Research - Dr. Nelson R. Manohar</title>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<div id=content>
<div id=title> <h1> <img src=./../HPS_INPUTS/REPORT/IMAGES/cat.jpg height=80 alt=hpslogo /> <span> HPS DNA MINING SYSTEMS RESEARCH - <a href=HPS_FOREWORD.HTM> <span class=green> DR. NELSON R.
MANOHAR </span> </a> </span> </h1> <span id=slogan> HPS DNA MINING SYSTEMS (c) 2007 BY NELSON R. MANOHAR-ALERS, PH.D.  </span> </div>

<div id=menu>
<div class=submit>
<ul>
<li><a href=HPS_FOREWORD.HTM class=unselected>                 <span><b>FOREWORD</b></span></a></li>
<li><a href=HPS_INTRODUCTION.HTM class=selected>             <span><b>INTRO</b></span></a></li>
<li><a href=HPS_INPUTS.HTM class=unselected>                   <span><b>INPUTS</b></span></a></li>
<li><a href=HPS_DATA_DESCRIPTION.HTM class=unselected>         <span><b>OPTIONAL</b></span></a></li>
<li><a href=HPS_BEHAVIOR.HTM class=unselected>                 <span><b>PERF</b></span></a></li>
<li><a href=HPS_MATCHING_INSTANCES.HTM class=unselected>       <span><b>MATCHES</b></span></a></li>
<li><a href=HPS_RECONSTRUCTION_OPS.HTM class=unselected>       <span><b>REPAIR</b></span></a></li>
<li><a href=HPS_SIMILARITY_PLOTS.HTM class=unselected>         <span><b>SIMILARITY</b></span></a></li>
<li><a href=HPS_PHILOGRAM_PLOT.HTM class=unselected>           <span><b>PHILOGRAM</b></span></a></li>
<li><a href=HPS_MOTIF_SEARCH.HTM class=unselected>             <span><b>MOTIFS</b></span></a></li>
<li><a href=HPS_MOTIF_REGEXPR.HTM class=unselected>            <span><b>MOTIF-MINER</b></span></a></li>
<li><a href=HPS_ABOUT_US.HTM class=unselected>                 <span><b>ABOUT</b></span></a></li>
<li><a href=HPS_DISCLAIMER.HTM class=unselected>               <span><b>DISCLAIMER</b></span></a></li>
</ul>
</div>
</div>


<div id=subheader>
<div class=rside>
Auto-generated report for multiple sequence alignment (MSA) analysis of the DNA signature against DNA test sequence.
</div>

<div class=lside>
<strong> Welcome! </strong> 
</div>
</div>

<div id=maincontent>
<A HREF=http://www3.webng.com/nelsonmanohar/research.htm> <P align=center> <B> <EM> (c) Nelson R. Manohar, Ph.D. - 2005-2007. All rights are reserved. </EM> </B> </span> </A> 

<div id=right_side>
<div class=lcontent align=justify> <h3> SITE MAP FOR AUTO-GENERATED REPORT</h3> 
<BR>   <a href=HPS_FOREWORD.HTM>                 <span><b>FOREWORD</b></span></a> 	 - important remarks.</li>
<BR>   <a href=HPS_INTRODUCTION.HTM>             <span><b>INTRODUCTION</b></span></a>    - to HPS DNA Mining Systems.</li>
<BR>   <a href=HPS_INPUTS.HTM>                   <span><b>INPUTS</b></span></a> 	 - description of the processed inputs.</li>
<BR>   <a href=HPS_DATA_DESCRIPTION.HTM>         <span><b>OPTIONAL</b></span></a> 	 - user-provide description of inputs.</li>
<BR>   <a href=HPS_BEHAVIOR.HTM>                 <span><b>PERFORMANCE</b></span></a>     - performance of dna mining.</li>
<BR>   <a href=HPS_MATCHING_INSTANCES.HTM>       <span><b>MATCHES</b></span></a>         - overview of matching instances.</li>
<BR>   <a href=HPS_RECONSTRUCTION_OPS.HTM>       <span><b>RECONSTRUCTION</b></span></a>  - summary of matching instances.</li>
<BR>   <a href=HPS_SIMILARITY_PLOTS.HTM>         <span><b>SIMILARITY</b></span></a>      - before/after dot matrix plots.</li>
<BR>   <a href=HPS_PHILOGRAM_PLOT.HTM>           <span><b>PHILOGRAM</b></span></a>       - signature-rooted philogram.</li>
<BR>   <a href=HPS_MOTIF_SEARCH.HTM>             <span><b>MOTIF-ID</b></span></a>        - motif characterization tool.</li>
<BR>   <a href=HPS_MOTIF_REGEXPR.HTM>            <span><b>MOTIF-MINER</b></span></a>     - motif miner tool.</li>
<BR>   <a href=HPS_ABOUT_US.HTM>                 <span><b>ABOUT USR</b></span></a>       - about us.</li>
<BR>   <a href=HPS_DISCLAIMER.HTM>               <span><b>DISCLAIMER</b></span></a>      - important remarks.</li>
</div>
</div>

<div id=left_side align=justify>



<H3>AUTO-GENERATED REPORT FOR SIMILARITY SEARCH </H3>
<P> 	This <B> auto-generated </B> report presents results of a <B> sequence aligning and phylogenetic 
	mining search. </B> Given a DNA signature and a DNA test sequence, "HPS DNA mining systems" are capable of 
	correctly identifying, extracting, and aligning ALL (that is, zero or more) TRUE matching 
	instances of said DNA signature found to be within said DNA test sequence.
	We define a <B> true matching instance </B> to be a subsequence of the DNA test sequence for 
	which either of the following (necessary and sufficient) criteria is met:
	(1) <B> the subsequence is an EXACT replica (100% compatible) of the DNA signature </B> or
	(2) <B> the subsequence exhibits TOLERABLE accumulation of DNA match errors WHILE retaining
	        certain identifying qualities.</B>  
	Specifically, accumulation of error may take place in any of the following ways:
	(2a) a matching instance exhibits a tolerable accumulation of <B> SPORADIC SINGLE-POINT 
		BASE-PAIR MUTATIONS </B> (such as inversions, transcription errors, duplications, etc.),
	(2b) a matching instance exhibits tolerable error bursts of <B> DELETED base-pairs </B>
		in either DNA signature or DNA test sequence,
	(2c) a matching instance exhibits tolerable error bursts of <B> INSERTED base-pairs </B> in 
		either DNA signature or the DNA test sequence, or
	(2d) a matching instance exhibits tolerable accumulation of error due to 
		<B> ANY COMBINATION OF THE ABOVE. </B>
	The details of how data mining parameters are selected by "HPS DNA mining systems" are purposely curtailed. 
	Suffice to say that the user needs only provide the DNA signature and the DNA test sequence and nothing more.  
	"HPS DNA mining systems" will then examine potential matching instances of the DNA signature found within 
	the DNA test sequence and reports true, optimal, and feasible matching instances found within.  
</P> 

<H4>NEW RUN-TIME COMPLEXITY LOWER BOUNDS </H4>
<P> 	More importantly, "HPS DNA mining systems" complete said tasks within NEW AND LOWER RUN-TIME COMPLEXITY BOUNDS 
	previously considered not feasible in practical cases and general use. 
	For example, "HPS DNA mining systems" can find matching 
	instances of a DNA signature of size <B> M </B> within a sequence of size <B> N </B> in sub-linear 
	time with respect to the input size <B> N. </B> That is, <B> "HPS DNA mining systems" can perform 
	<em> (certain types of) </em> COMBINATORIAL DATA MINING over sequences of size N in SUB-LINEAR TIME! </B>
	As a matter of fact, in practice (and regardless of input size), linear run-time cost operations 
	(such as time-series reading/writing) do take more time than the run-time cost for the "HPS DNA mining systems" 
	data mining core which (as stated) performs a COMBINATORIAL ANALYSIS of the entire DNA sequence. 
	Those versed in the arts, will recognize this to be an extraordinary result of vast implications.
</P> 
<H4>PHYLOGENETIC AND MUTATION SEARCH </H4>
<P> 	Moreover, <B> given R potentially phylogenetic DNA sequences,</B> by simply selecting (subsequence content) 
	from any one such as the DNA signature,  "HPS DNA mining systems" make possible to <B> find true matching instances 
	from the remaining R-1 </B> (potentially phylogenetic or damaged) DNA sequences in optimal time.
	Matching instances unearthed by "HPS DNA mining systems" are guaranteed to be true instances of the given 
	DNA signature, this being true EVEN when said matching instances may exhibit substantial DNA damage 
	(for example, due to mutations, deletions, inversions, insertions, duplications, transcription errors).
	Those versed in the arts will recognize that such error cases typically complicate correct identification 
	by inducing probabilities of misidentifications as well as resulting in significantly larger (such as 
	sub-quadratic as opposed to sub-linear) run-time computational costs in existing alternative approaches 
	in use today.  Moreover, "HPS DNA mining systems" not only unearth and identify true matching instances 
	but also produce rich and detailed output which identifies 
		(1) all <B> DIFFERENCES </B> between a DNA signature and any such matching instance 
		(2) as well as all <B> REPAIRS </B> (e.g., the logical placement of necessary DELETES 
		    and/or INSERTS operations) needed to transform (or repair) any 
	            (<B>potentially phylogenetic or damaged</B>) matching instance into the DNA 
		    signature (and vice versa). 
	For example, "HPS DNA mining systems" produce alignment-edit graphs, similarity graphs,
	base-to-base comparison reports, graphical identification of matching instances, and other 
	not yet disclosed analysis reports and graphs, etc.
</P> 
<H4>DATABASE AND WEB MINING </H4>
<P> 	Additionally, "HPS DNA mining systems" allow the search for matching instances of the given DNA signature 
	to be performed against a DNA sequence database (whether such is remotely located (as in the case of ENTREZ) 
	or stored on your hard-disk. Moreover, database sequences can be accessed in either the ENTREZ format or 
	in a time series format ((genomic-address, base) tuples).  Moreover, "HPS DNA mining systems" are 
	capable of automatically extracting time-series data from DNA sequence data found in ENTREZ webpages. 
</P>
<H4>DEMONSTRATION DATABASE </H4>
<P>	For demonstration purposes, "HPS DNA mining systems" have been tested on the complete Escherichia coli 
	K12 bacterium genome (that is, a genome of approximately 4.7 million base pairs (bp) and close to 
	400 DNA (10,000+ bp) DNA fragments).  The Escherichia coli K12 bacteria genome (U00096) - by 
	    <EM> Blattner,F.R., Plunkett,G. III, Bloch,C.A., Perna,N.T., Burland,V.,
            	 Riley,M., Collado-Vides,J., Glasner,J.D., Rode,C.K., Mayhew,G.F., 
	    	 Gregor,J., Davis,N.W., Kirkpatrick,H.A., Goeden,M.A., Rose,D.J., Mau,B. and Shao,Y.,
	    	 from "The complete genome sequence of Escherichia coli K-12", Science 277 (5331), 1453-1474 (1997) 
	    </EM> - was obtained from ENTREZ.
</P> 
</P> 	<P align=center> <B> <EM> (c) Nelson R. Manohar, Ph.D. - 2005-2007. All rights are reserved. </EM> </B> </span> </P> 
</BODY> </HTML> 



</div>
</div>

<div id=footer>
<p class=right> NELSON R. MANOHAR &copy; 2007  &middot; Updated on 5/07 &middot; Design: 
<a href=http://www.free-css-templates.com title=Designed by free-css-templates.com>free-css-templates.com</a> &middot;
<a href=HPS_DISCLAIMER.HTM target=_blank>License Terms</a> 
</p>
</div> 
</div> </body> </html>
