<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><meta name="robots" content="index, nofollow, noarchive"/><title>NCBI Sequence Viewer v2.0</title><!--MUTABLE--><!--www.ncbi.nlm.nih.gov:80--><!--MUTABLE--><link type="text/css" rel="stylesheet" href="http://www.ncbi.nlm.nih.gov/corehtml/ncbi_test.css"/><link type="text/css" rel="stylesheet" href="../sviewer/viewer.css"/><script type="text/javascript" src="../sviewer/viewer.js"> </script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/coreweb/javascript/popupmenu2/popupmenu2_6loader.js"> </script></head><body ><form name="frmQueryBox0" action="../entrez/query.fcgi" method="get" style="margin:0"><table width="100%" border="0" cellpadding="0" cellspacing="0"><tr><td><table width="100%" border="0" cellpadding="0" cellspacing="0"><tr><td align="left" width="130"><a href="http://www.ncbi.nlm.nih.gov"><img src="http://www.ncbi.nlm.nih.gov/corehtml/left.GIF" width="130" height="45" border="0" alt="NCBI"/></a></td><td align="left"><img src="/entrez/query/static/gifs/entrez_nuc.gif" alt="Nucleotide banner"/></td><td><table class="medium1" border="0" bordercolor="#336699" cellpadding="2" cellspacing="0" align="right"><tr><td bgcolor="#336699" align="right"><a href="http://www.ncbi.nlm.nih.gov/entrez/cubby.fcgi?call=QueryExt.CubbyQuery..ShowAll&amp;db=nucleotide"><font color="#FFFFFF"><b>My NCBI</b></font></a></td></tr><tr><td nowrap="1"><!--MUTABLE--><a href="http://www.ncbi.nlm.nih.gov/entrez/login.fcgi?call=so.SignOn..Login&amp;currentURL=http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?&amp;cmd=Display&amp;dopt=gbwithparts&amp;ef_CDD=8&amp;ef_Exon=512&amp;ef_HPRD=32&amp;ef_MGC=16&amp;ef_STS=64&amp;ef_microRNA=256&amp;ef_tRNA=128&amp;extrafeat=1016&amp;from=2185&amp;linkbar=jsmenu3&amp;list_uids=1786217&amp;to=3753" onClick="if(myncbi_xo == 0){window.open('http://www.ncbi.nlm.nih.gov/entrez/login.fcgi?call=so.SignOn..Login&amp;amp;tmpl=MiniBaseFile','Save2Cubby','resizable=yes,scrollbars=yes,toolbar=no,location=no,directories=no,status=no,menubar=no,copyhistory=no,alwaysRaised=yes,depend=yes,top=50,left=200,width=400,height=100');return false;} return true;"><!--MUTABLE-->[Sign In]</a> 
                                <!--MUTABLE--><a href="http://www.ncbi.nlm.nih.gov/entrez/login.fcgi?call=so.SignOn..Register&amp;currentURL=http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?&amp;cmd=Display&amp;dopt=gbwithparts&amp;ef_CDD=8&amp;ef_Exon=512&amp;ef_HPRD=32&amp;ef_MGC=16&amp;ef_STS=64&amp;ef_microRNA=256&amp;ef_tRNA=128&amp;extrafeat=1016&amp;from=2185&amp;linkbar=jsmenu3&amp;list_uids=1786217&amp;to=3753" onClick="if(myncbi_xo == 0){window.open('http://www.ncbi.nlm.nih.gov/entrez/login.fcgi?call=so.SignOn..Register&amp;amp;tmpl=MiniBaseFile','Save2Cubby','resizable=yes,scrollbars=yes,toolbar=no,location=no,directories=no,status=no,menubar=no,copyhistory=no,alwaysRaised=yes,depend=yes,top=50,left=200,width=400,height=100');return false;} return true;" title="Click to register for an account"><!--MUTABLE-->[Register]</a></td></tr></table></td></tr></table></td></tr><tr><td><table class="text" border="0" cellspacing="0" cellpadding="2" bgcolor="#000000" width="100%"><tr class="text" align="center"><td><a href="/entrez/query.fcgi?db=PubMed" class="gutter3"><font color="#ffffff">PubMed</font></a></td><td><a href="/entrez/query.fcgi?db=Nucleotide" class="gutter3"><font color="#ffffff">Nucleotide</font></a></td><td><a href="/entrez/query.fcgi?db=Protein" class="gutter3"><font color="#ffffff">Protein</font></a></td><td><a href="/entrez/query.fcgi?db=Genome" class="gutter3"><font color="#ffffff">Genome</font></a></td><td><a href="/entrez/query.fcgi?db=Structure" class="gutter3"><font color="#ffffff">Structure</font></a></td><td><a href="/entrez/query.fcgi?db=PMC" class="gutter3"><font color="#ffffff">PMC</font></a></td><td><a href="/entrez/query.fcgi?db=Taxonomy" class="gutter3"><font color="#ffffff">Taxonomy</font></a></td><td><a href="/entrez/query.fcgi?db=OMIM" class="gutter3"><font color="#ffffff">OMIM</font></a></td><td><a href="/entrez/query.fcgi?db=Books" class="gutter3"><font color="#ffffff">Books</font></a></td></tr></table></td></tr><tr><td><table width="100%" border="0" cellspacing="0" cellpadding="1" bgcolor="#cccccc"><tr><td class="small1" nowrap="1">
                                             Search <small><select name="db" title="Select database for search"><option value="pubmed">PubMed</option><option value="protein">Protein</option><option value="nucleotide" selected="1">Nucleotide</option><option value="nuccore">  CoreNucleotide</option><option value="nucgss">  GSS</option><option value="nucest">  EST</option><option value="structure">Structure</option><option value="genome">Genome</option><option value="books">Books</option><option value="cancerchromosomes">CancerChromosomes</option><option value="cdd">Conserved Domains</option><option value="domains">3D Domains</option><option value="gene">Gene</option><option value="genomeprj">Genome Project</option><option value="gensat">GENSAT</option><option value="geo">GEO Profiles</option><option value="gds">GEO Datasets</option><option value="homologene">HomoloGene</option><option value="journals">Journals</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="snp">SNP</option><option value="taxonomy">Taxonomy</option><option value="unigene">UniGene</option><option value="unists">UniSTS</option></select></small><input name="cmd" type="hidden" value=""/> for
                                            <input name="term" size="45" type="text" value=""/> 
                                            <input name="go" type="submit" value="Go" title="Send search request"/> 
                                            <input name="clear" type="button" value="Clear" onclick="this.form.term.value='';this.form.term.focus();" title="Clear search field"/></td></tr></table></td></tr></table><noscript><b><big><font color="#ff0000">You need JavaScript to work with this page.</font></big></b></noscript><table border="0" bgcolor="#cccccc" cellspacing="0" cellpadding="3" width="100%"><tr><td class="tabna" width="50"> </td><td nowrap="1" width="14%"><a CLASS="pmlinkna" HREF="javascript:Go('Limits')">Limits</a></td><td CLASS="tabna" NOWRAP="1" WIDTH="14%"><a CLASS="pmlinkna" HREF="javascript:Go('Index')">Preview/Index</a></td><td CLASS="tabna" NOWRAP="1" WIDTH="14%"><a CLASS="pmlinkna" HREF="javascript:Go('History')">History</a></td><td CLASS="tabna" NOWRAP="1" WIDTH="14%"><a CLASS="pmlinkna" HREF="javascript:Go('Clipboard')">Clipboard</a></td><td CLASS="tabna" NOWRAP="1" WIDTH="14%"><a CLASS="pmlinkna" HREF="javascript:Go('Details')">Details</a></td><td CLASS="tabna" NOWRAP="1" WIDTH="14%"/></tr></table></form><form action="viewer.fcgi" method="get" name="frmViewerBox" style="margin:0"><table bgcolor="#c0c0c0" border="0" cellpadding="4" cellspacing="0" width="100%"><tr><td nowrap="yes"><input name="db" type="hidden" value="nucleotide"/><input name="qty" type="hidden" value="1"/><input name="c_start" type="hidden" value="1"/><input id="list_uids" name="list_uids" type="hidden" value="1786217"/><input id="uids" name="uids" type="hidden" value=""/><small> Display 
                                <select name="dopt" title="Choose format of representation" onchange="submit();"><option value="gb">GenBank</option><option value="gbwithparts" selected="1">GenBank(Full)</option><option value="gbc">INSDSeq XML</option><option value="fasta">FASTA</option><option value="tinyseq">TinySeq XML</option><option value="asn">ASN.1</option><option value="xml">XML</option><option value="graph">Graph</option><option value="gi">GI List</option><option value="brief">Brief</option><option value="docsum">Summary</option></select>
                                 Show
                                <select name="dispmax" onchange="submit()"><option value="1">1</option><option value="2">2</option><option value="5" selected="1">5</option><option value="10">10</option><option value="20">20</option><option value="50">50</option><option value="100">100</option><option value="200">200</option><option value="500">500</option></select><select name="sendto" onchange="do_sendto(this); submit();"><option value="" selected="1">Send to</option><option value="t">Text</option><option value="on">File</option><option value="Add To Clipboard">Clipboard</option></select>
                                     
                                    <span class="group">
                                         Hide: 
                                        <input type="hidden" id="fmt_mask" name="fmt_mask" value="0"/><label title="Hide sequence and translations"><input type="checkbox" id="truncate" name="truncate" value="294912"/>
                                              sequence
                                        </label>
                                         
                                        <label title="Hide features such as variations, CDD regions, CDS products, repeat_regions etc"><input type="checkbox" id="less_feat" name="less_feat" value="504"/>
                                              all but gene, CDS and mRNA features
                                        </label>
                                         
                                    </span></small></td></tr></table><div style="background-color:#c0c0c0" width="100%"><table bgcolor="#c0c0c0" border="0" cellpadding="4" cellspacing="0"><tr><td nowrap="1"><small>
                          <span class="group" title="Show only region of the sequence (from-to)">Range: from 
                        <input type="text" name="from" size="9" maxlength="12" value="2185"/>
                         to 
                        <input type="text" name="to" size="9" maxlength="12" value="3753"/>
                             <input type="button" value="Show whole sequence" onclick="to.value='begin'; from.value='end'; submit();"/></span><label for="strand"> 
                            <span class="group" title="Choose strand to project data to"><input type="checkbox" id="strand" name="strand"/>
                                    Reverse complemented strand
                            </span></label></small></td><td nowrap="1"><small><input type="hidden" name="extrafeatpresent" value="1"/><span class="group" title="Mark external annotations to include in report">Features:
                
                  <input type="button" value="+" title="Show full list of annotations" onclick="operate_passive_features(this)"/><span id="passive_features" style="display:none"> 
                    <label id="ef_SNP"><input type="checkbox" id="ef_SNP" name="ef_SNP" value="1"/><span class="defeat" title="No annotation of Variations">SNP</span></label> 
                    <label id="ef_CDD"><input type="checkbox" id="ef_CDD" name="ef_CDD" value="8" checked="1"/><span class="defeat" title="No annotation of Conserved Domains">CDD</span></label> 
                    <label id="ef_MGC"><input type="checkbox" id="ef_MGC" name="ef_MGC" value="16" checked="1"/><span class="defeat" title="No annotation of Differences vs. the Reference Genome">MGC</span></label> 
                    <label id="ef_HPRD"><input type="checkbox" id="ef_HPRD" name="ef_HPRD" value="32" checked="1"/><span class="defeat" title="No annotation of Post-Translational Modifications from HPRD">HPRD</span></label> 
                    <label id="ef_STS"><input type="checkbox" id="ef_STS" name="ef_STS" value="64" checked="1"/><span class="defeat" title="No annotation of Sequence Tagged Site">STS</span></label> 
                    <label id="ef_tRNA"><input type="checkbox" id="ef_tRNA" name="ef_tRNA" value="128" checked="1"/><span class="defeat" title="No annotation of tRNAs (predicted)">tRNA</span></label> 
                    <label id="ef_microRNA"><input type="checkbox" id="ef_microRNA" name="ef_microRNA" value="256" checked="1"/><span class="defeat" title="No annotation of microRNA">microRNA</span></label> 
                    <label id="ef_Exon"><input type="checkbox" id="ef_Exon" name="ef_Exon" value="512" checked="1"/><span class="defeat" title="No annotation of Exon, or the location on a transcript that is derived from an exon">Exon</span></label></span></span></small></td><td><input type="submit" value="Refresh" id="refresh"/></td></tr></table></div><script type="text/javascript">
                var fmt_mod  = new FormatModifer();
            </script></form>
<div class='records'><!-- docsumok 1786217 1786217 1 0 0 -->
<div class='docsum' id='gi_1786217'>
<table width='100%' border='0' cellpadding='0' cellspacing='0' >
<tr>
<td><input name="uid" type="checkbox" value="1786217"><b>1: </b>&nbsp;<a href='/entrez/viewer.fcgi?db=nucleotide&val=1786217'>AE000114</a>.&nbsp;<a class="dblinks" href="javascript:PopUpMenu2_Set(Reports1786217)" onMouseOut="PopUpMenu2_Hide()">Reports<script language="JavaScript1.2">
<!--
var PopUpMenu2_LocalConfig_Reports1786217 = [
  ["TitleText","Reports"]
]
var Reports1786217 = [
  ["UseLocalConfig","Reports1786217","",""],
  ["ASN.1","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=asn'","",""],
  ["XML","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=xml'","",""],
  ["Summary","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=docsum'","",""],
  ["Brief","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=brief'","",""],
  ["FASTA","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=fasta'","",""],
  ["TinySeq XML","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=fasta_xml'","",""],
  ["GenBank","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=gb'","",""],
  ["INSDSeq XML","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=gbc_xml'","",""],
  ["GenBank(Full)","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=gbwithparts'","",""],
  ["GI List","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=gi'","",""],
  ["Graphic","window.top.location='/entrez/viewer.fcgi?list_uids=1786217&db=nucleotide&dopt=graph'","",""],
  ["Revision History","window.top.location='/entrez/sutils/girevhist.cgi?val=AE000114'","",""]
]
//-->
</script>
</a>&nbsp;...[gi:1786217] 
<font color='red'>The record has been replaced by <a href='/entrez/viewer.fcgi?db=nucleotide&val=U00096'>U00096</a></font></td>
<td align='right' width='15%'></td>
</tr>
</table>
</div>
<div class='recordbody'><div class="sequence"><a name="locus_1786217"></a><div class="localnav"><ul class="locals"><li><a href="#comment_1786217" title="Jump to the comment section of this record">Comment</a></li><li><a href="#feature_1786217" title="Jump to the feature table of this record">Features</a></li><li><a href="#sequence_1786217" title="Jump to the sequence of this record">Sequence</a></li></ul></div>
<pre class="genbank">LOCUS       AE000114                1569 bp    DNA     linear   BCT 01-DEC-2000
DEFINITION  Escherichia coli K12 MG1655 section 4 of 400 of the complete
            genome.
ACCESSION   <a href=/entrez/viewer.fcgi?val=AE000114>AE000114</a> REGION: 2185..3753
VERSION     AE000114.1  GI:1786217
KEYWORDS    .
SOURCE      Escherichia coli K12
  ORGANISM  <a href=http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=83333>Escherichia coli K12</a>
            Bacteria; Proteobacteria; Gammaproteobacteria; Enterobacteriales;
            Enterobacteriaceae; Escherichia.
REFERENCE   1  (bases 1 to 1569)
  AUTHORS   Blattner,F.R., Plunkett,G. III, Bloch,C.A., Perna,N.T., Burland,V.,
            Riley,M., Collado-Vides,J., Glasner,J.D., Rode,C.K., Mayhew,G.F.,
            Gregor,J., Davis,N.W., Kirkpatrick,H.A., Goeden,M.A., Rose,D.J.,
            Mau,B. and Shao,Y.
  TITLE     The complete genome sequence of Escherichia coli K-12
  JOURNAL   Science 277 (5331), 1453-1474 (1997)
   PUBMED   <a href=/sites/entrez?cmd=Retrieve&db=pubmed&list_uids=9278503>9278503</a>
REFERENCE   2  (bases 1 to 1569)
  AUTHORS   Blattner,F.R.
  TITLE     Direct Submission
  JOURNAL   Submitted (16-JAN-1997) Guy Plunkett III, Laboratory of Genetics,
            University of Wisconsin, 445 Henry Mall, Madison, WI 53706, USA.
            Email: ecoli@genetics.wisc.edu Phone: 608-262-2534 Fax:
            608-263-7459
REFERENCE   3  (bases 1 to 1569)
  AUTHORS   Blattner,F.R.
  TITLE     Direct Submission
  JOURNAL   Submitted (02-SEP-1997) Guy Plunkett III, Laboratory of Genetics,
            University of Wisconsin, 445 Henry Mall, Madison, WI 53706, USA.
            Email: ecoli@genetics.wisc.edu Phone: 608-262-2534 Fax:
            608-263-7459
REFERENCE   4  (bases 1 to 1569)
  AUTHORS   Plunkett,G. III.
  TITLE     Direct Submission
  JOURNAL   Submitted (13-OCT-1998) Laboratory of Genetics, University of
            Wisconsin, 445 Henry Mall, Madison, WI 53706, USA
<a name="comment_1786217"></a>COMMENT     [WARNING] On Jun 21, 2004 this sequence was replaced by
            gi:<a href=/entrez/viewer.fcgi?val=48994873>48994873</a>.
            This sequence was determined by the E. coli Genome Project at the
            University of Wisconsin-Madison (Frederick R. Blattner, director).
            Supported by NIH grants HG00301 and HG01428 (from the Human Genome
            Project and NCHGR). The entire sequence was independently
            determined from E. coli K12 strain MG1655. Predicted open reading
            frames were determined using GeneMark software, kindly supplied by
            Mark Borodovsky, Georgia Institute of Technology, Atlanta, GA,
            30332 [e-mail: mark@amber.gatech.edu].  Open reading frames that
            have been correlated with genetic loci are being annotated with CG
            Site Nos., unique ID nos. for the genes in the E. coli Genetic
            Stock Center (CGSC) database at Yale University, kindly supplied by
            Mary Berlyn. A public version of the database is accessible
            (<a href=http://cgsc.biology.yale.edu>http://cgsc.biology.yale.edu</a>). Annotation of the genome is an
            ongoing task whose goal is to make the genome sequence more useful
            by correlating it with other data.  Comments to the authors are
            appreciated. Updated information will be available at the E. coli
            Genome Project's World Wide Web site
            (<a href=http://www.genetics.wisc.edu>http://www.genetics.wisc.edu</a>). *** The E. coli K12 sequence and
            its annotations are periodically updated; this is version M54. No
            sequence changes. Annotation updates: updated gene identifications
            and products; all new functional assignments courtesy of Monica
            Riley; added promoters, protein binding sites, and repeated
            sequences described in reference 1. The unique numeric identifiers
            beginning with a lowercase 'b' assigned to each gene (protein- or
            RNA-encoding) are now designated as gene synonyms instead of
            labels. This should allow them to be searched for in Entrez as gene
            names.
<a name="feature_1786217"></a>FEATURES             Location/Qualifiers
     source          1..1569
                     /organism="Escherichia coli K12"
                     /mol_type="genomic DNA"
                     /strain="K12"
                     /sub_strain="MG1655"
                     /db_xref="taxon:<a href=http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=83333>83333</a>"
     gene            complement(1..1569)
                     /gene="caiC"
                     /note="synonym: b0037"
     CDS             complement(1..1569)
                     /gene="caiC"
                     /EC_number="<a href=http://www.chem.qmw.ac.uk/iubmb/enzyme/>6.3.2.-</a>"
                     /function="putative enzyme; Central intermediary
                     metabolism: Pool, multipurpose conversions"
                     /note="f522; 100 pct identical to CAIC_ECOLI SW: P31552"
                     /codon_start=1
                     /transl_table=<a href=http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c#SG11>11</a>
                     /product="probable crotonobetaine/carnitine-CoA ligase"
                     /protein_id="<a href=/entrez/viewer.fcgi?val=AAC73148.1>AAC73148.1</a>"
                     /db_xref="GI:1786221"
                     /translation="MDRGAMDIIGGQHLRQMWDDLADVYGHKTALICESSGGVVNRYS
                     YLELNQEINRTANLFYTLGIRKGDKVALHLDNCPEFIFCWFGLAKIGAIMVPINARLL
                     CEESAWILQNSQACLLVTSAQFYPMYQQIQQEDATQLRHICLTDVALPADDGVSSFTQ
                     LKNQQPATLCYAPPLSTDDTAEILFTSGTTSRPKGVVITHYNLRFAGYYSAWQCALRD
                     DDVYLTVMPAFHIDCQCTAAMAAFSAGATFVLVEKYSARAFWGQVQKYRATVTECIPM
                     MIRTLMVQPPSANDQQHRLREVMFYLNLSEQEKDAFCERFGVRLLTSYGMTETIVGII
                     GDRPGDKRRWPSIGRVGFCYEAEIRDDHNRPLPAGEIGEICIKGIPGKTIFKEYFLNP
                     QATAKVLEADGWLHTGDTGYRDEEDFFYFVDRRCNMIKRGGENVSCVELENIIAAHPK
                     IQDIVVVGIKDSIRDEAIKAFVVLNEGETLSEEEFFRFCEQNMAKFKVPSYLEIRKDL
                     PRNCSGKIIRKNLK"
     protein_bind    complement(1558..1567)
                     /gene="caiC"
                     /note="No predicted promoter"
                     /bound_moiety="MetJ predicted site"
ORIGIN      
<a name="sequence_1786217"></a>        1 ttatttcaga ttctttctaa ttattttccc cgagcaatta cgtggcagat cttttctgat
       61 ctccagataa gagggcactt taaatttcgc catattttgt tcgcagaagc ggaaaaattc
      121 ctcttcgctc aatgtttcac cttcattcag caccacaaat gctttgatgg cttcatcgcg
      181 aatcgaatct ttaataccca caaccacgat gtcctgaatt ttcgggtgcg cggcgataat
      241 attttccagc tccacgcagg agacattctc gccgccacgt ttaatcatat tgcagcggcg
      301 atcgacgaaa taaaaaaagt cctcttcgtc gcggtatccg gtatcgccgg tatgcagcca
      361 gccatcggct tccagcactt tcgcagtggc ttgtgggttg agaaagtact ctttgaagat
      421 ggttttccca ggtatgcctt taatgcagat ttcaccgatc tcaccagccg ggagcgggcg
      481 attgtgatcg tcgcggatct ccgcttcgta gcaaaacccc acccgaccaa tcgacggcca
      541 gcgtcgttta tcgccaggac gatcgccgat aatgcccaca atggtttccg tcatcccata
      601 agacgtcagc aagcgaacgc cgaagcgttc acaaaacgca tctttttcct gctccgacaa
      661 gttgagataa aacatcactt cccgcaggcg gtgttgctga tcgttcgctg aaggcggctg
      721 taccatcaac gtacggatca tcatcggaat acattcggta acggtggcgc ggtacttctg
      781 tacctgtccc cagaaggcgc gggcgctgta tttctcgacc agcacaaagg tggccccggc
      841 agaaaacgcc gccatcgccg cagtacactg gcaatcgata tgaaacgcag gcattaccgt
      901 caggtagacg tcatcgtcac gcagtgcaca ctgccaggcg gagtaatatc cagcgaagcg
      961 caggttgtaa tgggtaatca ccacaccttt cggtcgggag gtggtgccgg aggtgaagag
     1021 aatttccgcc gtatcgtcag tcgatagcgg cggtgcatag cacaaggtgg caggttgttg
     1081 atttttcagt tgagtaaacg aactcacgcc atcatcagcg ggaagtgcca catctgtcag
     1141 gcaaatgtgc cgcaattgag tggcatcttc ctgctgaatc tgttgataca taggatagaa
     1201 ttgcgcactg gtcaccagca ggcacgcctg gctattttgc aggatccacg cgctttcctc
     1261 gcacaacagg cgggcgttaa tcggcaccat aatcgcgcca atttttgcca gcccgaacca
     1321 gcaaaagata aattccgggc agttgtcgag atgtagtgca accttgtcgc ctttgcgaat
     1381 ccccagcgta taaaacaggt ttgccgtgcg gttaatctcc tgatttaact caagataact
     1441 ataccggtta acgactccgc cgctggattc acaaatcagc gccgttttat gaccgtaaac
     1501 gtccgcaaga tcgtcccaca tttgacgtag atgttgtccg ccaatgatat ccattgcacc
     1561 tctatccat
//</pre>
<a name="slash_1786217"></a></div></div>
</div><div align="center" class="medium1"><p><a href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a> |
                <a href="mailto:info@ncbi.nlm.nih.gov">Write to the Help Desk</a><br/><a href="http://www.ncbi.nlm.nih.gov">NCBI</a> |
                <a href="http://www.nlm.nih.gov">NLM</a> |
                <a href="http://www.nih.gov">NIH</a></p><p> </p></div><p class="dblinks" align="right"><font color="#eeeeee" size="-5"><!--MUTABLE-->Aug 28 2007 16:53:42<!--MUTABLE--></font></p></body></html>
