
<!DOCTYPE html PUBLIC -//W3C//DTD XHTML 1.0 Strict//EN http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd>
<html xmlns=http://www.w3.org/1999/xhtml xml:lang=en lang=en>
<head>
<meta http-equiv=content-type content=text/html;charset=iso-8859-2 />
<meta name=author content=free-css-templates.com />
<link rel=stylesheet href=./../HPS_INPUTS/REPORT/stylesheet.css type=text/css />
<title>HPS Research - Dr. Nelson R. Manohar</title>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<div id=content>
<div id=title> <h1> <img src=./../HPS_INPUTS/REPORT/IMAGES/cat.jpg height=80 alt=hpslogo /> <span> HPS DNA MINING SYSTEMS RESEARCH - <a href=HPS_FOREWORD.HTM> <span class=green> DR. NELSON R.
MANOHAR </span> </a> </span> </h1> <span id=slogan> HPS DNA MINING SYSTEMS (c) 2007 BY NELSON R. MANOHAR-ALERS, PH.D.  </span> </div>

<div id=menu>
<div class=submit>
<ul>
<li><a href=HPS_FOREWORD.HTM class=unselected>                 <span><b>FOREWORD</b></span></a></li>
<li><a href=HPS_INTRODUCTION.HTM class=unselected>             <span><b>INTRO</b></span></a></li>
<li><a href=HPS_INPUTS.HTM class=unselected>                   <span><b>INPUTS</b></span></a></li>
<li><a href=HPS_DATA_DESCRIPTION.HTM class=unselected>         <span><b>OPTIONAL</b></span></a></li>
<li><a href=HPS_BEHAVIOR.HTM class=selected>                 <span><b>PERF</b></span></a></li>
<li><a href=HPS_MATCHING_INSTANCES.HTM class=unselected>       <span><b>MATCHES</b></span></a></li>
<li><a href=HPS_RECONSTRUCTION_OPS.HTM class=unselected>       <span><b>REPAIR</b></span></a></li>
<li><a href=HPS_SIMILARITY_PLOTS.HTM class=unselected>         <span><b>SIMILARITY</b></span></a></li>
<li><a href=HPS_PHILOGRAM_PLOT.HTM class=unselected>           <span><b>PHILOGRAM</b></span></a></li>
<li><a href=HPS_MOTIF_SEARCH.HTM class=unselected>             <span><b>MOTIFS</b></span></a></li>
<li><a href=HPS_MOTIF_REGEXPR.HTM class=unselected>            <span><b>MOTIF-MINER</b></span></a></li>
<li><a href=HPS_ABOUT_US.HTM class=unselected>                 <span><b>ABOUT</b></span></a></li>
<li><a href=HPS_DISCLAIMER.HTM class=unselected>               <span><b>DISCLAIMER</b></span></a></li>
</ul>
</div>
</div>


<div id=subheader>
<div class=rside>
Auto-generated report for multiple sequence alignment (MSA) analysis of the DNA signature against DNA test sequence.
</div>

<div class=lside>
<strong> Welcome! </strong> 
</div>
</div>

<div id=maincontent>
<A HREF=http://www3.webng.com/nelsonmanohar/research.htm> <P align=center> <B> <EM> (c) Nelson R. Manohar, Ph.D. - 2005-2007. All rights are reserved. </EM> </B> </span> </A> 

<div id=right_side>
<div class=lcontent align=justify> <h3> SITE MAP FOR AUTO-GENERATED REPORT</h3> 
<BR>   <a href=HPS_FOREWORD.HTM>                 <span><b>FOREWORD</b></span></a> 	 - important remarks.</li>
<BR>   <a href=HPS_INTRODUCTION.HTM>             <span><b>INTRODUCTION</b></span></a>    - to HPS DNA Mining Systems.</li>
<BR>   <a href=HPS_INPUTS.HTM>                   <span><b>INPUTS</b></span></a> 	 - description of the processed inputs.</li>
<BR>   <a href=HPS_DATA_DESCRIPTION.HTM>         <span><b>OPTIONAL</b></span></a> 	 - user-provide description of inputs.</li>
<BR>   <a href=HPS_BEHAVIOR.HTM>                 <span><b>PERFORMANCE</b></span></a>     - performance of dna mining.</li>
<BR>   <a href=HPS_MATCHING_INSTANCES.HTM>       <span><b>MATCHES</b></span></a>         - overview of matching instances.</li>
<BR>   <a href=HPS_RECONSTRUCTION_OPS.HTM>       <span><b>RECONSTRUCTION</b></span></a>  - summary of matching instances.</li>
<BR>   <a href=HPS_SIMILARITY_PLOTS.HTM>         <span><b>SIMILARITY</b></span></a>      - before/after dot matrix plots.</li>
<BR>   <a href=HPS_PHILOGRAM_PLOT.HTM>           <span><b>PHILOGRAM</b></span></a>       - signature-rooted philogram.</li>
<BR>   <a href=HPS_MOTIF_SEARCH.HTM>             <span><b>MOTIF-ID</b></span></a>        - motif characterization tool.</li>
<BR>   <a href=HPS_MOTIF_REGEXPR.HTM>            <span><b>MOTIF-MINER</b></span></a>     - motif miner tool.</li>
<BR>   <a href=HPS_ABOUT_US.HTM>                 <span><b>ABOUT USR</b></span></a>       - about us.</li>
<BR>   <a href=HPS_DISCLAIMER.HTM>               <span><b>DISCLAIMER</b></span></a>      - important remarks.</li>
</div>
</div>

<div id=left_side align=justify>



<HTML> <HEAD> <TITLE>HPS DNA MINING SYSTEMS</TITLE> <STYLE>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 


<HTML> <HEAD> <TITLE>HPS DNA MINER</TITLE> 
<STYLE>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<H3> BEHAVIOR OF THE MINING PROCESS ACROSS TIME </H3>
<P> 	The graphs below show the behavior of the data mining process over time as it operates in the
        multidimensional space of the transformed data. The bottom graph on the following figure shows 
	a time plot of the behavior of the mining process in term of the observed match (feasibility) 
	strength signal as it gets evaluated across increasing genomic addresses of the DNA input 
	sequence. A combinatorial data mining is performed yet in sub-linear time with respect to the
	size of the DNA input sequence. This data mining always operates at a fraction of the linear 
	run-time cost associated with sequential operations such as either I/O-bound (i.e., reading 
	and writing) of the DNA input sequence or CPU-bound operations over the entire DNA input sequence.
	Moreover, this high-performance data mining is combinatorial in nature, that is, it exhaustively
	examines ALL possible match opportunities. More importantly, the behavior of the combinatorial
	data-mining process is well-behaved; that is, even when extracted by said high-performance 
	combinatorial data mining engine, matching instances exhibit an extraordinary signal-to-noise 
	ratio over non-matching instances.   
	The upper graph shows a histogram of the resultant match strenght signal which in turn reveals 
	a GAUSSIAN fit coupled with a HEAVY-TAIL outlier presence (i.e., LOGNORMAL fit in the formal 
	sense).  That is, under this well-behaved mining process, matching instances represent 
	HEAVY-TAIL outliers with respect to the general population of values which behaves as Gaussian noise. 
</P> 
<BR>
<BR>
<BR>
<BR>



<TABLE WIDTH=1200 BGCOLOR=#E6F2FF ALIGN=JUSTIFY>
<TH> HISTOGRAM OF MATCH RATING SIGNAL VALUES ACROSS ALL PIVOT POINTS IN THE DNA TEST SEQUENCE</TH>
<TR> <TD> <IMG SRC=DNA_MATCH_BEHAVIOR_HISTOGRAM.PNG WIDTH=1200 HEIGHT=800 HSPACE=0 VSPACE=0 BORDER=0 ALIGN=JUSTIFY> </TD> </TR>
</TABLE>

<TABLE WIDTH=1200 BGCOLOR=#E6F2FF> <TR> <TD AlIGN=TOP> <SPAN STYLE='font-size:9.0pt;font-family:Verdana'> <DL> 
<DT> <STRONG>TOP:</STRONG> Histogram of the match strength signal with search region, white noise, and matching instances identified.</DT>
<DT> <STRONG>TOP: GREEN-FIT</STRONG> Resultant gaussian curve for mean and sigma parameters derived from the data (when heavy tail outliers are included).  </DT> 
<DT> <STRONG>TOP: BLUE-FIT</STRONG> Best normal curve fit for resultant mean and sigma parameters (when heavy tail ooutliers are removed).  </DT> 
<DT> <STRONG>BOTTOM</STRONG> Time plot of the match strength signal in the nth-dimensional optimization space, with matching instances identified. </DT> 
</DL> </SPAN> </TD> </TR> </TABLE>
</BODY> </HTML>


</BODY> </HTML> 


<BR>

<h2 class=underline> About us (www_nelsonmanohar_us) </h2> <img src=./../HPS_INPUTS/REPORT/IMAGES/bookcase.jpg alt=aboutus/>
<p> <strong>Dr. Manohar-Alers</strong> earned a B.S. in computer engineering in 1987. 
He also earned a M.S. computer engineering from the University of Wisconsin at Madison in 1988 and a M.S.E. in 
industrial engineering from the University of Michigan at Ann Arbor in 1992.  Dr. Manohar-Alers earned his Ph.D. 
in computer science and engineering specializing in software systems from the University of Michigan at Ann Arbor 
in 1997.  Previously, he worked at the IBM T. J. Watson Research Center as a Research Staff Member and at AT&T 
Bell Laboratories (now Lucent Bell Labs) as a Member of the Technical Staff.  His interests are in experimental 
software systems dealing with scalable systems theory, measurements, and resource management.  He has been lead 
inventor in several patents on resource management.  Previous research related to groupware, multimedia authoring, 
and advanced intelligent networks.  Dr. Manohar-Alers' credentials are found on his <a href=vitae/2008-NRM_CV.pdf 
target=_blank> <b> curriculum vitae. </b></a> He currently works as an independent researcher in Puerto Rico - 
USA, where Dr. Manohar-Alers has threaded his long-term research onto applied research on <strong>state estimation 
for random processes.</strong> This research introduces the <strong>Harmonic Process State (HPS) transform</strong>.
The <strong>HPS Transform</strong> exhibits desirable qualities on implementation ease, algorithmic complexity, 
computational stability, signal compressibility, decision-making robustness, information loss, and error behavior. 
Although the <strong>HPS Transform</strong> is particularly suited for process control, the results have vast 
implications to other fields.  You may contact Dr. Manohar-Alers at <a href=NRM_Contact.htm target=_blank> 
<strong>nelsonmanohar@yahoo.com</strong>. </a> </p>

<h2>OUR ASSETS</h2>
<p> Dr. Manohar-Alers is currently unaffiliated to a research institution or grant. His body of research work has 
been done independently, in a self-supported manner, and thus readily available for use at a future research affiliation.
His research work has the potential to produce high-impact (theoretical and applied) assets related to research 
publications, software, and patents of particular targeting toward valuable application domains.  </p>

<h2>CONTACT US</h2>
<p> This website is chiefly made available toward principals from research and academic institutes expressly evaluating 
the credentials of Dr. Manohar-Alers for employment within their institutes.  Partly because of time-zone differences, 
e-mail is preferred in contacting Dr. Manohar-Alers.  </p>

<h2>DISCLAIMER</H2>
<p> No data is collected (by this website) during your visit to this website.  This website may provide access to its 
author's copyrighted material, intellectual property, and/or privileged information for expressly confidential review.
We ask that you respect our copyrights and confidentiality requests.  This website may contain documents filed pursuant 
USC-compliant claims entered with the USDOJ.  This website does not intentionally support nor contain illegal nor 
unlawful content.  Please read important information about this website's license terms <a href=NRM_license.htm> 
(i.e., <b>TERMS-OF-USE</b>). </a> </p>

</div>
</div>

<div id=footer>
<p class=right> NELSON R. MANOHAR &copy; 2007  &middot; Updated on 5/07 &middot; Design: 
<a href=http://www.free-css-templates.com title=Designed by free-css-templates.com>free-css-templates.com</a> &middot;
<a href=HPS_DISCLAIMER.HTM target=_blank>License Terms</a> 
</p>
</div> 
</div> </body> </html>
