
<!DOCTYPE html PUBLIC -//W3C//DTD XHTML 1.0 Strict//EN http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd>
<html xmlns=http://www.w3.org/1999/xhtml xml:lang=en lang=en>
<head>
<meta http-equiv=content-type content=text/html;charset=iso-8859-2 />
<meta name=author content=free-css-templates.com />
<link rel=stylesheet href=./../HPS_INPUTS/REPORT/stylesheet.css type=text/css />
<title>HPS Research - Dr. Nelson R. Manohar</title>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<div id=content>
<div id=title> <h1> <img src=./../HPS_INPUTS/REPORT/IMAGES/cat.jpg height=80 alt=hpslogo /> <span> HPS DNA MINING SYSTEMS RESEARCH - <a href=HPS_FOREWORD.HTM> <span class=green> DR. NELSON R.
MANOHAR </span> </a> </span> </h1> <span id=slogan> HPS DNA MINING SYSTEMS (c) 2007 BY NELSON R. MANOHAR-ALERS, PH.D.  </span> </div>

<div id=menu>
<div class=submit>
<ul>
<li><a href=HPS_FOREWORD.HTM class=unselected>                 <span><b>FOREWORD</b></span></a></li>
<li><a href=HPS_INTRODUCTION.HTM class=unselected>             <span><b>INTRO</b></span></a></li>
<li><a href=HPS_INPUTS.HTM class=unselected>                   <span><b>INPUTS</b></span></a></li>
<li><a href=HPS_DATA_DESCRIPTION.HTM class=unselected>         <span><b>OPTIONAL</b></span></a></li>
<li><a href=HPS_BEHAVIOR.HTM class=unselected>                 <span><b>PERF</b></span></a></li>
<li><a href=HPS_MATCHING_INSTANCES.HTM class=unselected>       <span><b>MATCHES</b></span></a></li>
<li><a href=HPS_RECONSTRUCTION_OPS.HTM class=selected>       <span><b>REPAIR</b></span></a></li>
<li><a href=HPS_SIMILARITY_PLOTS.HTM class=unselected>         <span><b>SIMILARITY</b></span></a></li>
<li><a href=HPS_PHILOGRAM_PLOT.HTM class=unselected>           <span><b>PHILOGRAM</b></span></a></li>
<li><a href=HPS_MOTIF_SEARCH.HTM class=unselected>             <span><b>MOTIFS</b></span></a></li>
<li><a href=HPS_MOTIF_REGEXPR.HTM class=unselected>            <span><b>MOTIF-MINER</b></span></a></li>
<li><a href=HPS_ABOUT_US.HTM class=unselected>                 <span><b>ABOUT</b></span></a></li>
<li><a href=HPS_DISCLAIMER.HTM class=unselected>               <span><b>DISCLAIMER</b></span></a></li>
</ul>
</div>
</div>


<div id=subheader>
<div class=rside>
Auto-generated report for multiple sequence alignment (MSA) analysis of the DNA signature against DNA test sequence.
</div>

<div class=lside>
<strong> Welcome! </strong> 
</div>
</div>

<div id=maincontent>
<A HREF=http://www3.webng.com/nelsonmanohar/research.htm> <P align=center> <B> <EM> (c) Nelson R. Manohar, Ph.D. - 2005-2007. All rights are reserved. </EM> </B> </span> </A> 

<div id=right_side>
<div class=lcontent align=justify> <h3> SITE MAP FOR AUTO-GENERATED REPORT</h3> 
<BR>   <a href=HPS_FOREWORD.HTM>                 <span><b>FOREWORD</b></span></a> 	 - important remarks.</li>
<BR>   <a href=HPS_INTRODUCTION.HTM>             <span><b>INTRODUCTION</b></span></a>    - to HPS DNA Mining Systems.</li>
<BR>   <a href=HPS_INPUTS.HTM>                   <span><b>INPUTS</b></span></a> 	 - description of the processed inputs.</li>
<BR>   <a href=HPS_DATA_DESCRIPTION.HTM>         <span><b>OPTIONAL</b></span></a> 	 - user-provide description of inputs.</li>
<BR>   <a href=HPS_BEHAVIOR.HTM>                 <span><b>PERFORMANCE</b></span></a>     - performance of dna mining.</li>
<BR>   <a href=HPS_MATCHING_INSTANCES.HTM>       <span><b>MATCHES</b></span></a>         - overview of matching instances.</li>
<BR>   <a href=HPS_RECONSTRUCTION_OPS.HTM>       <span><b>RECONSTRUCTION</b></span></a>  - summary of matching instances.</li>
<BR>   <a href=HPS_SIMILARITY_PLOTS.HTM>         <span><b>SIMILARITY</b></span></a>      - before/after dot matrix plots.</li>
<BR>   <a href=HPS_PHILOGRAM_PLOT.HTM>           <span><b>PHILOGRAM</b></span></a>       - signature-rooted philogram.</li>
<BR>   <a href=HPS_MOTIF_SEARCH.HTM>             <span><b>MOTIF-ID</b></span></a>        - motif characterization tool.</li>
<BR>   <a href=HPS_MOTIF_REGEXPR.HTM>            <span><b>MOTIF-MINER</b></span></a>     - motif miner tool.</li>
<BR>   <a href=HPS_ABOUT_US.HTM>                 <span><b>ABOUT USR</b></span></a>       - about us.</li>
<BR>   <a href=HPS_DISCLAIMER.HTM>               <span><b>DISCLAIMER</b></span></a>      - important remarks.</li>
</div>
</div>

<div id=left_side align=justify>



<HTML> <HEAD> <TITLE>HPS DNA MINING SYSTEMS</TITLE> <STYLE>
<!--
@font-face {font-family:Helvetica; panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face {font-family:Courier; panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face {font-family:Times; panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:10.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:10.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 

<H3> RECONSTRUCTION OF BURST DAMAGED DNA (OR PHYLOGENETIC) MATCHING INSTANCES </H3>
<P>  	Matching instances unearthed by "HPS DNA mining systems" may actually be (mutated as well as potentially phylogenetic) 
	versions of the given DNA signature, which nevertheless retain some special intrinsic qualities associated with 
	the DNA signature.
	Typically, in such cases, special operations are used to show how, for any matching instance said to represent 
	a damaged or mutated version of the DNA signature, it is possible to (optimally) reconstruct (i.e. with few
	operations and few changes) the targeted DNA signature sequence.
</P>
<H4> RECONSTRUCTION OPERATIONS </H4>
<P> 	The minimal abstract base machine needed to perform such reconstruction consists of just two operations: 
	<B> DELETE(X) </B> and <B> INSERT(X).</B>  The <B> DELETE(X) </B> operation edits a DNA sequence (at 
	the current genomic address) by deleting  the next <B> X bases </B> from it.  The <B> INSERT(X) </B> 
	operation edits a DNA sequence (at the current genomic address) by inserting <B> X base placeholders </B> 
	into it.  A <B> DELETE/INSERT </B> operation is triggered when a damage burst of more than <B> Z bases </B> 
	is observed within a DNA sequence.  Suppose that such event is detected at genomic address <B> Y.</B> 
	Then, such triggering event will be identified by the <B> RESYNC (Y, Z)</B>  operation, which is 
	therefore always followed by either a <B> DELETE(X) </B> or <B> INSERT(X) </B> operation.
	Finally, we choose to use another couple of operations to identify the exacting genomic address 
	of any matching instance.  First, a <B> PRELIM-ALIGN(Y) </B> operation marks the preliminary 
	genomic address believed to be the start of a matching instance.
	Then, an <B> OFFSET-ALIGN(W) </B> operation identifies how far away (in number of bases, 
	in either direction) is the exacting genomic address that corresponds to the start of the 
	matching instance. Note that (as stated), both these genomic addresses refer to indexing within 
	the DNA input sequence, which consists of the DNA signature PREPENDED to the DNA test sequence).
	As stated, to obtain the true genomic address (MSTART) for the start  of a matching instance, 
	one need only to refer to the DETAILED REPORT page for said matching instance or alternatively, 
	subtract the size <B> M </B> of the DNA signature from the offset-corrected preliminary addresses. 
	That is, <B> MSTART=Y+W-M. </B>
</P>
<H4> ADJUSTED AND ORIGINAL PAIRINGS </H4>
<P> 	Finally, to differentiate between original and reconstructed cases, we refer to the pairing of 
	the DNA signature to the ORIGINAL (i.e., as found) matching instance M# from the DNA test 
	sequence to as the <B> ORIGINAL SIG-TO-M# pairing. </B> Similarly, we refer to the pairing 
	of the DNA signature to the ADJUSTED (i.e., after reconstruction operations) matching instance M# 
	to as the <B> ADJUSTED SIG-TO-M# pairing. </B>  The following figures show, for each matching 
	instance extracted from the DNA test sequence, the resultant reconstruction operations needed 
	to recover the DNA signature from said matching instance.  For this reason, each figure has 
	two parts. Whereas the top part of each figure shows the  <B> ADJUSTED SIG-TO-M# pairing </B> 
	together with the various reconstruction operations needed, the bottom part of each figure 
	shows the <B> ORIGINAL SIG-TO-M# pairing </B> of DNA signature to matching instance (as given). 
	Within any pairing plot, the DNA signature is always colored in "light-blue" color whereas 
	the matching instance is always colored in "light-pink".  
</P>
<H4> BEHAVIOR OF THE RECONSTRUCTION </H3>
<P> 	Note that when a matching instance exhibits no burst DNA damage, no reconstruction needs 
	to be apply (i.e., all DNA damage is due to single-point errors) and thus, ORIGINAL and ADJUSTED 
	pairings are the same.  Moreover, note that when a matching instance exhibits burst DNA damage, 
	while the ORIGINAL pairing shows clear cross-correspondence discrepancies between DNA signature 
	and matching instance due to the misalignment induced by such burst DNA damage, after such 
	burst DNA damage is corrected via reconstruction operations, the resultant ADJUSTED pairing 
	exhibits consistently accurate cross-correspondence.  Note however, that while burst DNA damage 
	is corrected during reconstruction, single point random errors are always left (on either pairing) 
	uncorrected (such showing in various test-cases). To this end, note that for each matching instance 
	a summary statistics tuple is also given, which shows the number of mismatched base pairs in the
	cross-correspondence check between the DNA signature and the ADJUSTED (reconstructed) matching instance.
	Such reported mismatches are due to single-point random errors plus the repair cost (in bases) of 
	reconstruction operations.
</P> 
</BODY> </HTML>
<TABLE WIDTH=1200 BGCOLOR=#E6F2FF> 
<TR> <TD ALIGN=TOP> <SPAN STYLE='font-size:9.0pt;font-family:Verdana'> <DL> <TR><TD> 

<HTML> 
<HEAD> <TITLE>BEST MATCHING INSTANCES OF DNA SIGNATURE WITHIN TEST SEQUENCE </TITLE> <STYLE>
<!--
@font-face {font-family:Verdana; panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face {font-family:Garamond; panose-1:2 2 4 4 3 3 1 1 8 3;}
p.MsoNormal, li.MsoNormal, div.MsoNormal {margin:0in; margin-bottom:.0001pt; font-size:8.0pt; font-family:Verdana;}
@page Section0 {size:8.5in 11.0in; margin:0.5in 0.5in 0.5in 0.5in; font-size:8.0pt; font-family:Verdana;}
div.Section0 {page:Section0;}
-->
</STYLE> </HEAD> <BODY LANG=EN-US> 
<DIV CLASS=MSONORMAL ALIGN=center> 	


<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M0.HTM TARGET=_BLANK> M0:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 5999 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 6985 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.73170733 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 987 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 906.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M0.PNG> <BR><BR>
<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M1.HTM TARGET=_BLANK> M1:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 14467 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 15443 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.61538464 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 977 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 1221.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 34 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M1.PNG> <BR><BR>
<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M2.HTM TARGET=_BLANK> M2:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 19971 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 20961 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.5609756 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 991 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 1272.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 34 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M2.PNG> <BR><BR>
<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M3.HTM TARGET=_BLANK> M3:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 17988 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 18910 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.45 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 923 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 1268.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 64 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M3.PNG> <BR><BR>
<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M4.HTM TARGET=_BLANK> M4:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 33017 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 33952 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.44444445 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 936 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 1416.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 68 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M4.PNG> <BR><BR>
<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M5.HTM TARGET=_BLANK> M5:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 24981 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 25975 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.41463414 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 995 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 1603.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 68 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M5.PNG> <BR><BR>
<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M6.HTM TARGET=_BLANK> M6:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 8013 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 8978 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.3902439 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 966 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 1706.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 42 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M6.PNG> <BR><BR>
<BR> 
<H3> <A HREF=HPS_DNA_BESTMATCHING_PATTERNS_DETAILED_M7.HTM TARGET=_BLANK> M7:  DETAILED REPORT FOR MATCHING INSTANCE </A> </H3>
<TABLE ALIGN=CENTER>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_STARTS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> AND_ENDS_AT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_RATING </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_SPANS </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> MATCH_WEIGHT </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> NONMATCHBP </P> </TD>  
</TR>
<TR>
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 31007 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 31980 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 0.375 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 974 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 1272.0 </P> </TD> 
<TD WIDTH=120 VALIGN=TOP ALIGN=CENTER> <P CLASS=MSONORMAL> 106 </P> </TD>  
</TR>
</TABLE>
<BR>
<IMG SRC=HPS_DNA_ALIGNMENT_ADJUSTMENT_LINEPLOT_M7.PNG> <BR><BR>
 
<P ALIGN=CENTER> <B> <EM> HPS DNA mining systems (c) 2005, 2006, 2007 - Dr. Nelson R. Manohar. All rights reserved. </EM> </B> </P>
</DIV>
</BODY> 
</HTML> 
 </TD> </TR> </TABLE> 
</BODY> </HTML> 


<BR>

<h2 class=underline> About us (www_nelsonmanohar_us) </h2> <img src=./../HPS_INPUTS/REPORT/IMAGES/bookcase.jpg alt=aboutus/>
<p> <strong>Dr. Manohar-Alers</strong> earned a B.S. in computer engineering in 1987. 
He also earned a M.S. computer engineering from the University of Wisconsin at Madison in 1988 and a M.S.E. in 
industrial engineering from the University of Michigan at Ann Arbor in 1992.  Dr. Manohar-Alers earned his Ph.D. 
in computer science and engineering specializing in software systems from the University of Michigan at Ann Arbor 
in 1997.  Previously, he worked at the IBM T. J. Watson Research Center as a Research Staff Member and at AT&T 
Bell Laboratories (now Lucent Bell Labs) as a Member of the Technical Staff.  His interests are in experimental 
software systems dealing with scalable systems theory, measurements, and resource management.  He has been lead 
inventor in several patents on resource management.  Previous research related to groupware, multimedia authoring, 
and advanced intelligent networks.  Dr. Manohar-Alers' credentials are found on his <a href=vitae/2008-NRM_CV.pdf 
target=_blank> <b> curriculum vitae. </b></a> He currently works as an independent researcher in Puerto Rico - 
USA, where Dr. Manohar-Alers has threaded his long-term research onto applied research on <strong>state estimation 
for random processes.</strong> This research introduces the <strong>Harmonic Process State (HPS) transform</strong>.
The <strong>HPS Transform</strong> exhibits desirable qualities on implementation ease, algorithmic complexity, 
computational stability, signal compressibility, decision-making robustness, information loss, and error behavior. 
Although the <strong>HPS Transform</strong> is particularly suited for process control, the results have vast 
implications to other fields.  You may contact Dr. Manohar-Alers at <a href=NRM_Contact.htm target=_blank> 
<strong>nelsonmanohar@yahoo.com</strong>. </a> </p>

<h2>OUR ASSETS</h2>
<p> Dr. Manohar-Alers is currently unaffiliated to a research institution or grant. His body of research work has 
been done independently, in a self-supported manner, and thus readily available for use at a future research affiliation.
His research work has the potential to produce high-impact (theoretical and applied) assets related to research 
publications, software, and patents of particular targeting toward valuable application domains.  </p>

<h2>CONTACT US</h2>
<p> This website is chiefly made available toward principals from research and academic institutes expressly evaluating 
the credentials of Dr. Manohar-Alers for employment within their institutes.  Partly because of time-zone differences, 
e-mail is preferred in contacting Dr. Manohar-Alers.  </p>

<h2>DISCLAIMER</H2>
<p> No data is collected (by this website) during your visit to this website.  This website may provide access to its 
author's copyrighted material, intellectual property, and/or privileged information for expressly confidential review.
We ask that you respect our copyrights and confidentiality requests.  This website may contain documents filed pursuant 
USC-compliant claims entered with the USDOJ.  This website does not intentionally support nor contain illegal nor 
unlawful content.  Please read important information about this website's license terms <a href=NRM_license.htm> 
(i.e., <b>TERMS-OF-USE</b>). </a> </p>

</div>
</div>

<div id=footer>
<p class=right> NELSON R. MANOHAR &copy; 2007  &middot; Updated on 5/07 &middot; Design: 
<a href=http://www.free-css-templates.com title=Designed by free-css-templates.com>free-css-templates.com</a> &middot;
<a href=HPS_DISCLAIMER.HTM target=_blank>License Terms</a> 
</p>
</div> 
</div> </body> </html>
